packages/sdk/src/wallet/Ordit.ts

```diff
@@ -233,9 +235,12 @@ export class Ordit {
     return psbt.toHex()
   }
 
-  signMessage(message: string, type?: AddressFormats) {
+  signMessage(message: string, type?: AddressFormats, accountIndex?: number, addressIndex?: number) {
```

Comment:
can we add another params like `opts?` that will take in the optional params so its easier to add new opts next time without breaking. Then we can put `accountIndex` and `addressIndex` inside

Reply:
updated